conda/e3sm_diags_env.yml

```diff
@@ -5,19 +5,21 @@ channels:
   - e3sm
```

Comment:
should we remove e3sm channel since we are on conda-forge now?

Reply:
Yup, we can remove e3sm channel now. Thanks